### ISSUE TITLE
program blocked by youtube?

### ISSUE THREAD

**Issue by Ya-Ri** [rg3/youtube-dl#8834]
Hi,
am i wrong? but it seems that youtube has find a way to block the program...
The Ip address is no longuer blacklisted but the software is.
Do you work on it?
Regards,
YaRi

**Comment by phihag** [comment]
Sorry, but this bug report contains not nearly enough information. Please refer to our [guidelines for bug reporting](https://github.com/rg3/youtube-dl/blob/master/CONTRIBUTING.md) and file a new issue with the required information. Also note that extraordinary claims (such as that YouTube is actively blocking) require extraordinary evidence. It is far more likely that your ISP is meddling with traffic, YouTube is ratelimiting your IP, or you have an outdated version of youtube-dl than that YouTube has indeed starting blocking youtube-dl without any developer noting.

And to answer your question: youtube-dl is under active development, and works fine for me.
